sql/analyzer/fix_exec_indexes.go

```diff
@@ -102,6 +110,12 @@ func (s *idxScope) getIdx(n string) (int, bool) {
 			return i, true
 		}
 	}
+	n = unqualify(n)
+	for i := len(s.columns) - 1; i >= 0; i-- {
```

Comment:
can we not recurse on `getIdx(unqualify(n))`? documenting that this is only correct for references to SetOp outputs is also useful

Reply:
If unqualified `n` isn't in the scope, then I think that results in an infinite loop.